_docs/ops/security-ir.md

```diff
@@ -246,4 +246,4 @@ Guidelines for addressing Low-sev issues:
 
 * Last modified on: {% last_modified_at %}
 * [Recent document history](https://github.com/cloud-gov/cg-site/commits/master/{{ page.path }}) (since 2020-02-05)
-* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
\ No newline at end of file
+* [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
```

Comment:
Huh, wonder what got flagged as a change here...

Reply:
newline at the end of the file?